corehq/apps/cloudcare/views.py

```diff
@@ -114,14 +114,22 @@ def get_web_apps_available_to_user(self, domain, user):
         app_access = get_application_access_for_domain(domain)
         app_ids = get_app_ids_in_domain(domain)
 
+        def _can_access_web_app(user, domain, app):
+            if user.can_access_web_app(domain, app.get('copy_of', app.get('_id'))):
+                return True
+            if user.can_access_web_app(domain, app.get('upstream_app_id')):
```

Comment:
Can you elaborate on this one?  This is for the situation where there are two apps on the same domain, one linked to the other?  Would these show up as different entries on the roles page where you select which apps are accessible?